### ISSUE TITLE
Zip package contains __MACOSX folder

### ISSUE THREAD

**Issue by sethwoodworth** [tonsky/FiraCode#79]
```
$ cd ~/.local/share/fonts
$ unzip FiraCode_1.100.zip ./
$ tree
.
├── FiraCode-Bold.otf
├── FiraCode-Medium.otf
├── FiraCode-Regular.otf
├── FiraCode-Retina.otf
└── __MACOSX
```

**Comment by tonsky** [comment]
why is this an issue?

**Comment by Anachron** [comment]
It's useless? Would you like to have .desktop files in application code because the programmer is on Windows?

**Comment by tonsky** [comment]
I would not care. Anyways, fixed in 1.101
